deploy/crds/managed_v1alpha1_subjectpermission_cr.yaml

```diff
@@ -4,9 +4,6 @@ metadata:
   name: example-subjectpermission
 spec:
   subjectKind: Group
-  subjectName: Test
-  permissions:
```

Comment:
Why did you remove this?

Reply:
testing with ClusterPermission in this CR specifically. I have other CRs where I test permissions, and both together